data/ui/window.blp

```diff
@@ -525,18 +525,18 @@ template $GraphsWindow : Adw.ApplicationWindow {
 
 menu primary_menu {
   section{
-    item (_("Save Project"), "app.save_project")
-    item (_("Open Project"), "app.open_project")
+    item (_("Save Project…"), "app.save_project")
+    item (_("Open Project…"), "app.open_project")
   }
   section {
-    item (_("Export Data"), "app.export_data")
-    item (_("Export Figure"), "app.export_figure")
+    item (_("Export Data…"), "app.export_data")
+    item (_("Export Figure…"), "app.export_figure")
```

Comment:
I had to look up the use of ellipses in the Gnome HIG (the usage is correct here as far as I can tell), just for reference:

`Placeholders can be phrased in the same way as labels. Alternatively, if the text field performs an action, they can be phrased as an instruction, followed by ellipses. For example: “Search for a city…” or “Write a message…”. `
https://developer.gnome.org/hig/patterns/controls/text-fields.html#guidelines

Reply:
I was basically reading up on the HIGs comments on Wording and Text style, where I found this. Although the reference I used was [this one](https://developer.gnome.org/hig/guidelines/writing-style.html#ellipses), which states, that for each label, etc. that requires further action (selecting a file, adjusting parameters) an ellipsis should be used. Also note that this is the U+2026 Unicode Character instead of three dots.